assets/js/sidebar/sidebar-drawer.js

```diff
@@ -45,6 +48,7 @@ export function initialize () {
 export function update () {
   const pref = sessionStorage.getItem(SIDEBAR_STATE_KEY)
   const open = pref !== SIDEBAR_PREF_CLOSED && !isScreenSmall()
+  if (open) initializeList()
```

Comment:
This is the problematic line. It doesn't ever initialize the list on small screens. :) I will revert it on main but feel free to push a better fix if this should be maintained somehow.